ghcide/ghcide.cabal

```
@@ -260,7 +261,8 @@ executable ghcide
                 -rtsopts
                 -- disable idle GC
                 -- increase nursery size
-                "-with-rtsopts=-I0 -A128M"
+                -- Enable collection of heap statistics
+                "-with-rtsopts="-I0 -A128M -T"
```

I assume this got lost during rebase :smile: 
```suggestion
                "-with-rtsopts=-I0 -A128M -T"
```